src/complete.js

```diff
@@ -0,0 +1,19 @@
+import Storage from './storage.js';
+
+const tasksArray = JSON.parse(localStorage.getItem('todo')) || [];
+const complete = () => {
+  const check = document.querySelectorAll('input[type=checkbox]');
+  check.forEach((input, index) => input.addEventListener('change', () => {
+    if (input.checked) {
+      tasksArray[index].completed = true;
```

Comment:
- [ ] When we click on the checkbox kindly add a line through the checked item.

Reply:
this is optional not required